### PR TITLE
[IOTDB-719] add avg_series_point_number_threshold in config

### PR DESCRIPTION
**Pull request by qiaojialin**
When average series point number reaches a threshold, flush.

### CODE REVIEW

File: docs/UserGuide/Server/Config Manual.md [hunk 1]
```diff
@@ -272,6 +272,42 @@ The permission definitions are in ${IOTDB\_CONF}/conf/jmx.access.
...
+|Default| true |
+|Effective|After restart system|
+
+* memtable\_size\_threshold
+
+|Name| memtable\_size\_threshold |
+|:---:|:---|
+|Description| max memtable size|
+|Type|Long|
+|Default| false |
```
Comment (HTHou) [inline docs/UserGuide/Server/Config Manual.md:290]: Default value is false? I think it should be 1GB or 1073741824. 
```suggestion
|Type|Long|
|Default| 1073741824 |
```

File: docs/UserGuide/Server/Config Manual.md [hunk 2]
```diff
@@ -272,6 +272,42 @@ The permission definitions are in ${IOTDB\_CONF}/conf/jmx.access.
...
+|Default| 2048 |
+|Effective|After restart system|
+
+* tsfile\_size\_threshold
+
+|Name| tsfile\_size\_threshold |
+|:---:|:---|
+|Description| max tsfile size|
+|Type|Long|
+|Default| false |
```
Comment (HTHou) [inline docs/UserGuide/Server/Config Manual.md:308]:
```suggestion
|Type|Long|
|Default| 536870912 |
```

File: docs/zh/UserGuide/Server/Config Manual.md [hunk 3]
```diff
@@ -253,6 +253,42 @@
...
+|Default| true |
+|Effective|重启服务器生效|
+
+* memtable\_size\_threshold
+
+|Name| memtable\_size\_threshold |
+|:---:|:---|
+|Description| 内存缓冲区 memtable 阈值|
+|Type|Long|
+|Default| false |
```
Comment (HTHou) [inline docs/zh/UserGuide/Server/Config Manual.md:271]:
```suggestion
|Type|Long|
|Default| 1073741824 |
```

File: docs/zh/UserGuide/Server/Config Manual.md [hunk 4]
```diff
@@ -253,6 +253,42 @@
...
+|Default| 2048 |
+|Effective|重启服务器生效|
+
+* tsfile\_size\_threshold
+
+|Name| tsfile\_size\_threshold |
+|:---:|:---|
+|Description| 每个 tsfile 大小|
+|Type|Long|
+|Default| false |
```
Comment (HTHou) [inline docs/zh/UserGuide/Server/Config Manual.md:289]:
```suggestion
|Type|Long|
|Default| 536870912 |
```

File: docs/UserGuide/Server/Config Manual.md (hunk 1 above)
Comment (qiaojialin) [reply docs/UserGuide/Server/Config Manual.md:290]: fixed

File: docs/UserGuide/Server/Config Manual.md (hunk 2 above)
Comment (qiaojialin) [reply docs/UserGuide/Server/Config Manual.md]: fixed

File: docs/zh/UserGuide/Server/Config Manual.md (hunk 3 above)
Comment (qiaojialin) [reply docs/zh/UserGuide/Server/Config Manual.md]: fixed

File: docs/zh/UserGuide/Server/Config Manual.md (hunk 4 above)
Comment (qiaojialin) [reply docs/zh/UserGuide/Server/Config Manual.md]: fixed

**Pull request merged**